apps/cli/src/code-generator.spec.ts

```diff
@@ -0,0 +1,101 @@
+import {
+    Tree,
+    addProjectConfiguration,
+    joinPathFragments,
+    readProjectConfiguration,
+} from '@nx/devkit'
+import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing'
+import * as fs from 'fs/promises'
+import fetchMock from 'jest-fetch-mock'
+import * as path from 'path'
+
+import { CodeGeneratorOptionsTree, codeGenerator } from './code-generator'
+
+fetchMock.enableMocks()
```

Comment:
https://mswjs.io/ is my go to for mocking fetch.

Reply:
Moved over